metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -41,6 +45,8 @@ class GlossaryTermConfig(ConfigModel):
     owners: Optional[Owners]
     inherits: Optional[List[str]]
     contains: Optional[List[str]]
```

Comment:
let's add an additional Optional str field called `id`, which if present will be used as the id for the term. The id can be provided in `urn:li:..` form or in bare form.